dataland-dummy-qa-service/src/main/kotlin/org/dataland/datalanddummyqaservice/services/QaService.kt

```diff
@@ -0,0 +1,48 @@
+package org.dataland.datalanddummyqaservice.services
+
+import org.dataland.datalandmessagequeueutils.cloudevents.CloudEventMessageHandler
+import org.dataland.datalandmessagequeueutils.exceptions.MessageQueueException
+import org.slf4j.LoggerFactory
+import org.springframework.amqp.core.Message
+import org.springframework.amqp.rabbit.annotation.RabbitListener
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.stereotype.Component
+
+/**
+ * Implementation of a QA Service reacting on the upload_queue and forwarding message to qa_queue
+ * @param cloudEventMessageHandler service for managing CloudEvents messages
+ */
+@Component
+@ComponentScan("org.dataland")
```

Comment:
We used to have all @ComponentScan annotations on the main-kotlin file (i.e. the one where we also have @SpringBootApplication). 

We should stay consistent. here.

Reply:
Done

Reply:
There are 2 @ComponentScans missing - please remove them:
- ON DataManager
- On StorageController

Reply:
wip

Reply:
Compoentscans removed

Reply:
Sorry for beeing picky, but:
We do always have 
`@ComponentScan(basePackages = ["org.dataland"])`, except for the QaService, here we have 
`@ComponentScan("org.dataland")`

Let's be consistent.

Reply:
You are right, adapted it.